js.go

```diff
@@ -2485,6 +2488,14 @@ func ConsumerReplicas(replicas int) SubOpt {
 	})
 }
 
+// SetMemoryStorage sets the memory storage to true for a consumer.
+func SetMemoryStorage() SubOpt {
```

Comment:
To be more in line with other sub options, I would suggest changing the name to `MemoryStorage()`

Reply:
Yes. Also wonder when the name is "too generic", is going to be a problem if one day we need the same name for a different option? Meaning this is a SubOpt, but what if we need an option one day for JSOpt (or any other). For that reason, should we be more precise in the naming? Something like "ConsumerMemoryStorage()"? (not good, but you get the idea).

Reply:
I named it "SetMemoryStorage" because the name "MemoryStorage" is already being used. Should I name it "ConsumerMemoryStorage" as @kozlovic suggested?

Reply:
Yes please, go with ConsumerMemoryStorage() then, and this proves my point about possible name clashes unfortunately...